src/modules/market/components/market-basics/market-basics.jsx

```diff
@@ -58,7 +59,11 @@ const MarketBasics = (p) => {
         </h1>
 
         {(p.type === BINARY || p.type === SCALAR) &&
-          <MarketPreviewOutcomes outcomes={p.outcomes} min={p.minValue} max={p.maxValue} type={p.type} />
+          <MarketOutcomesBinaryScalar outcomes={p.outcomes} min={p.minValue} max={p.maxValue} type={p.type} />
```

Comment:
Generally you’ll want to line break attributes for readability if there are more than one.